### ISSUE TITLE
Suggestion: separate front ends from clients

### ISSUE THREAD

**Issue by lordkitsuna** [mendel5/alternative-front-ends#156]
I personally feel that "front end" is being used too loosely in the list. for example none of the twitch things are an actual web front end but just apps or clients. it would be nice if there was a separation between web front end, and alternative clients. I'm always on the search for actual front ends i can self host.

**Comment by meatspinner** [comment]
Yeah, the youtube section specifically is kind of a mess. Could separate it into, for instance, 3 sub-sections, which would be:

Web Frontends
Apps/Clients
Downloaders.

**Comment by maboroshin** [comment]
The three classifications by meatspinner are reclassified as follows:

* Desktop groups
* * Cross-platform groups if there are too many
* * Windows only groups if there are too many
* Mobile groups
* Andoroid only groups if there are too many (for Apps/Clients)
* * Andoroid TV at the end
* * (implicitly) iOS only groups continue 

* Youtube-DL groups if there are too many (for Downloaders)

**Comment by HookedBehemoth** [comment]
Websites fall somewhere between desktop and mobile.
Also splitting those would require e.g. having multiple twitter groups.

**Comment by ltguillaume** [comment]
You could just introduce "labels" for platforms, using simple colored tags saying Android/iOS/Web etc. instead of dividing everything into an endless amount of categories...
